zebra-network/src/peer_set/initialize.rs

```diff
@@ -478,6 +492,11 @@ where
             // Zebra can't control how many queued connections are waiting,
             // but most OSes also limit the number of queued inbound connections on a listener port.
             tokio::time::sleep(constants::MIN_PEER_CONNECTION_INTERVAL).await;
+        } else {
+            // Security: Let other tasks run after each errored connection is processed.
+            //
+            // Avoids remote peers starving other Zebra tasks using inbound connection errors.
+            tokio::task::yield_now().await;
         }
```

Comment:
Remove duplicate code: I made the yield unconditional in the PR I just merged.

So I'll just merge this fix now.
```suggestion
        }
```